tests/test_suite_common.sh

```diff
@@ -53,28 +58,30 @@ execute_tests_daemon() {
         run_individual_test 'mesh/ipip-cross-subnet'
         run_individual_test 'mesh/ipip-off'
     done
-    
-    # Turn the node-mesh off.  This will cause confd to terminate through one of it's own
+
+    # Turn the node-mesh off.  This will cause confd to terminate through one of its own
     # template updates.  Check that it does.
     turn_mesh_off
-    check_pid_exits $CONFD_PID
-
-    # Re-start a background confd.
-    echo "Restarting confd"
-    BGP_LOGSEVERITYSCREEN="debug" confd -confdir=/etc/calico/confd >$LOGPATH/logd2 2>&1 &
-    CONFD_PID=$!
+    # check_pid_exits $CONFD_PID
```

Comment:
It's correctly commented - because there is no longer the need for confd to exit and restart when nodeMeshEnabled changes - but very messy now, so I will clean this up!